db/migrate/20220916043854_delete_and_recreate_mhv_opt_in_flags.rb

```diff
@@ -0,0 +1,32 @@
+class DeleteAndRecreateMHVOptInFlags < ActiveRecord::Migration[6.1]
+  def up
+    drop_mhv_opt_in_flags_table
+    create_mhv_opt_in_flags_table
+  end
+
+  def down
```

Comment:
```suggestion
  def down
    drop_mhv_opt_in_flags_table
```

Reply:
For the 'down' you'll need to drop table too or it'll freak out when trying to recreate the table from the previous merge